plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraType.java

```diff
@@ -97,7 +98,10 @@ public List<CassandraType> getArgumentTypes()
 
     public String getName()
     {
-        return kind.name();
+        if (argumentTypes.size() == 0) {
+            return kind.name();
+        }
+        return format("%s<%s>", kind.name(), argumentTypes.stream().map(CassandraType::getName).collect(Collectors.joining(",")));
```

Comment:
This won't work in case of nested array. e.g. `array(array(int))`.

Reply:
You are right, it causes this error:
```
com.datastax.oss.driver.api.core.servererrors.InvalidQueryException: Non-frozen collections are not allowed inside collections: list<list<int>>
```
I'll try to fix it, perhaps by wrapping it with `frozen<>` where appropriate.